packages/pigeon/lib/dart_generator.dart

```diff
@@ -336,7 +348,7 @@ void _writeFlutterApi(
                   _addGenericTypesNullable(func.returnType);
               final bool isAsync = func.isAsynchronous;
               final String emptyReturnStatement = isMockHandler
-                  ? 'return <Object?, Object?>{};'
+                  ? 'return <Object?>[];'
```

Comment:
Do we use this, or is it just a dummy to make the mocks compile? Because this value would never actually be valid, right?